evap/evaluation/views.py

```diff
@@ -185,3 +185,28 @@ def set_lang(request):
         user.save()
 
     return set_language(request)
+
+
+@login_required
+def profile_edit(request):
+    user = request.user
+    if user.is_editor:
+        form = DelegatesForm(request.POST or None, request.FILES or None, instance=user)
+
+        if form.is_valid():
+            form.save()
+
+            messages.success(request, _("Successfully updated your profile."))
+            return redirect('evaluation:profile_edit')
+
+        return render(request, "profile.html", dict(
+            user=user,
+            form=form,
+            delegate_of=user.represented_users.all(),
+            cc_users=user.cc_users.all(),
+            ccing_users=user.ccing_users.all(),
+        ))
+
+    return render(request, "profile.html", dict(
```

Comment:
Codecov says this is not covered -- maybe add a simple test that simply loads this page